examples/nucleo_l476rg/lvgl/main.c

```diff
@@ -0,0 +1,154 @@
+#include <modm/board.hpp>
+#include <modm/debug.hpp>
+#include <modm/processing.hpp>
+#include <modm/driver/display/ili9341_spi.hpp>
+#include <modm/driver/touch/touch2046.hpp>
+
+#include <lvgl/lvgl.h>
+
+// USER INCLUDES
+#include "screens/test_screen.h"
+#include "screens/process_screen.h"
+
+
+// Set the log level
+#undef	MODM_LOG_LEVEL
+#define	MODM_LOG_LEVEL modm::log::DEBUG
+
+using namespace Board;
+using namespace modm::literals;
+
+
+namespace tft
+{
+	using DmaRx = Dma1::Channel2;
+	using DmaTx = Dma1::Channel3;
+	using Spi = SpiMaster1_Dma<DmaRx, DmaTx>;
+	//using Spi = SpiMaster1;
+	using Cs = Board::D10;
+	using Sck = Board::D13;
+	using Miso = Board::D12;
+	using Mosi = Board::D11;
+	using DataCommands = Board::D9;
+	using Reset = Board::D8;
+	using Backlight = modm::platform::GpioC9;
+}
+
+modm::Ili9341Spi<
+	tft::Spi,
+	tft::Cs,
+	tft::DataCommands,
+	tft::Reset,
+	tft::Backlight
+> tftController;
+
+namespace touch
+{
+	using Spi = SpiMaster3;
+	using Cs = modm::platform::GpioD2;
+	using Sck = modm::platform::GpioC10;
+	using Miso = modm::platform::GpioC11;
+	using Mosi = modm::platform::GpioC12;
+	using Interrupt = modm::platform::GpioC9;
+}
+
+modm::Touch2046<touch::Spi, touch::Cs> touchController;
+
+// Define display buffer 1
+static constexpr size_t buf_size = 240 * 320 / 8;
+static lv_color_t modm_aligned(4) buf[buf_size];
+
+// Define display buffer 2
+static constexpr size_t buf2_size = 240 * 320 / 8;
+static lv_color_t modm_aligned(4) buf2[buf2_size];
+
+void my_touchpad_read(lv_indev_t*, lv_indev_data_t* data)
+{
+	data->state = RF_CALL_BLOCKING(touchController.isTouched()) ? LV_INDEV_STATE_PRESSED : LV_INDEV_STATE_RELEASED;
```

Comment:
```suggestion
	data->state = touchController.isTouched() ? LV_INDEV_STATE_PRESSED : LV_INDEV_STATE_RELEASED;
```
Resumables are deprecated now